package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "globals",
-  "version": "8.12.0",
+  "version": "8.13.0",
```

Comment:
Don't bump versions in other people's projects ;)

I for one have a script for publishing which does this. You can't possible know which version I will pick either. I might want to do other changes with this one.